src/core/state-model.ts

```diff
@@ -53,6 +74,7 @@ export class UIPStateModel extends Observable {
     this.fire();
```

Comment:
Could we simplify the code below
```
 if ('transform' in cfg) {
      UIPStateModel.setAttribute(elements, attribute, cfg.transform);
    } else {
      UIPStateModel.setAttribute(elements, attribute, cfg.value);
    }
```
to 
`UIPStateModel.setAttribute(elements, attribute, 'transform' in cfg ? cfg.transform :  cfg.value);`